locales/sk/json.json

```diff
@@ -5,46 +5,46 @@
     "365 Days": "365 dní",
     "60 Days": "60 dní",
     "90 Days": "90 dní",
-    ":amount selected": ":Amount vybraných",
-    ":amount Total": ":Amount spolu",
-    ":days day trial": ":Days-dňová skúšobná verzia",
-    ":resource Details": ":Resource Podrobnosti",
-    ":resource Details: :title": ":Resource Podrobnosti: :title",
-    "A decryption key is required.": "A decryption key is required.",
-    "A fresh verification link has been sent to your email address.": "Na vašu e-mailovú adresu bol odoslaný nový overovací odkaz.",
-    "A new verification link has been sent to the email address you provided during registration.": "Na e-mailovú adresu, ktorú ste uviedli pri registrácii, bol odoslaný nový overovací odkaz.",
-    "A new verification link has been sent to the email address you provided in your profile settings.": "Na e-mailovú adresu, ktorú ste uviedli v nastaveniach profilu, bol odoslaný nový overovací odkaz.",
-    "A new verification link has been sent to your email address.": "Na vašu e-mailovú adresu bol odoslaný nový overovací odkaz.",
-    "Accept Invitation": "Prijať Pozvanie",
+    ":amount selected": "Vybraných :amount ",
```

Comment:
An extra space character at the end of a line.

Reply:
Removed